### PR TITLE
Add rpc api and pipectl command to get application

### LINKED ISSUE #1294: Add rpc api and pipectl command to get application
**What would you like to be added**:

**Why is this needed**:

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1294 

**Does this PR introduce a user-facing change?**:

```release-note
Add rpc api and pipectl command to get application
```

### CODE REVIEW

File: pkg/insightstore/BUILD.bazel [hunk 1]
```diff
@@ -4,31 +4,27 @@ go_library(
     name = "go_default_library",
     srcs = [
         "chunk.go",
-        "filestore.go",
         "filepath.go",
+        "filestore.go",
```
Comment (khanhtc1202) [inline pkg/insightstore/BUILD.bazel:8]: nits, should we remove this file from this PR, looks like it should be a part of another PR ( insight feature implementation ) 🤔

Comment (nghialv) [reply pkg/insightstore/BUILD.bazel:8]: Sure. This was generated by `make gazelle` but you are right. Let me remove them from this PR.

File: pkg/app/pipectl/cmd/application/get.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	if err != nil {
+		return fmt.Errorf("failed to get application: %w", err)
+	}
+
+	bytes, err := json.Marshal(resp.Application)
+	if err != nil {
+		return fmt.Errorf("failed to marshal application: %w", err)
+	}
+
+	fmt.Println(string(bytes))
```
Comment (nakabonne) [inline pkg/app/pipectl/cmd/application/get.go:71]: It may be better to let the `get` struct have `stdout` and be `fmt.Fprintln(c.stdout, string(bytes))` here. But it's not a problem to do when testing at all.

Comment (nghialv) [reply pkg/app/pipectl/cmd/application/get.go]: Nice. I have just added it.
